scripts/pages/home/view.mjs

```diff
@@ -0,0 +1,631 @@
+import { RAPID_API_MATCHES_URL_PARAMS } from "../../api/api.mjs";
+
+export default {
+
+    matchesTemplate: document.querySelector('.matches-table__template'),
+    matchesTable: document.getElementsByTagName('tbody')[0],
+    heroesTemplate: document.querySelector('.heroes-table__template'),
+    heroesTable: document.getElementsByTagName('tbody')[1],
+    body: document.getElementsByTagName('body')[0],
+    buttonToTop: document.querySelector('#to-top'),
+    buttonToTopMark: document.querySelector('.mark'),
+    openNavButton: document.getElementById('openbtn'),
+    profileAnchor: document.getElementById('profile'),
+    matchesAnchor: document.getElementById('matches'),
+    matchesSortButtonsArray: document.getElementsByClassName('matches-table__sort-button'),
+    heroesSortButtonsArray: document.getElementsByClassName('heroes-table__sort-button'),
+    heroesAnchor: document.getElementById('heroes'),
+    statisticsAnchor: document.getElementById('statistics'),
+    matchIdCopyArray: document.getElementsByClassName('text-copy'),
+
+    matchesLastPressedIndex: null,
+    heroesLastPressedIndex: null,
+
+    renderPlaceholders() {
+        for (let i = 0; i < 25; i++) {
+            this.matchesTable.append(this.matchesTemplate.content.cloneNode(true));
+            i < 8 ? this.heroesTable.append(this.heroesTemplate.content.cloneNode(true)) : null;
+        }
+    },
+
+    openNav() {
+        let navbar = document.querySelector('.navbar');
+        navbar.classList.toggle('active');
+        console.log("ok");
+    },
+
+    scrollToTop() {
+        document.body.scrollTop = 0;
+        document.documentElement.scrollTop = 0;
+    },
+
+    toTop(buttonToTop, buttonToTopMark) {
+        if (window.scrollY > 300) {
+            buttonToTop.classList.add('show');
+        } else {
+            buttonToTop.classList.remove('show');
+            buttonToTopMark.classList.remove('show');
+        }
+    },
+
+    copyIdToClipboard(button) {
+        let temp = document.createElement('textarea');
+        temp.style.position = "fixed";
+        temp.style.opcity = "0";
+        temp.textContent = button.getAttribute("data-label");
+        console.log(button.getAttribute("data-label"));
+
+        document.body.appendChild(temp);
+        temp.select();
+        document.execCommand('copy');
+        document.body.removeChild(temp);
+    },
+
+    pressButtonMatches(buttonIndex) {
+        console.log(buttonIndex, this.matchesLastPressedIndex);
+
+        this.matchesSortButtonsArray[buttonIndex].classList.add('pressed');
+
+        if (this.matchesLastPressedIndex !== null) {
+            this.matchesSortButtonsArray[this.matchesLastPressedIndex].classList.remove('pressed');
```

Comment:
There is `classList.toggle` API for that. So you don't have to store `matchesLastPressedIndex`